server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2020,4 +2111,23 @@ private void addCalculatedAttributes(Entitlement ent) {
         ent.getPool().setCalculatedAttributes(calculatedAttributes);
     }
 
+    private Consumer verifyExportParams(String consumerUuid, String cdnLabel) {
```

Comment:
if we split this into consumer validation and cdn validation, and move this from the resource class, we could save ourselves from some redundancy.